Cargo.toml

```
@@ -1,5 +1,5 @@
 [workspace]
-members = [".", "ipld/resolver", "testing/e2e", "identity"]
+members = [".", "ipld/resolver", "testing/e2e", "identity", "testing/child-subnet-infra"]
```

I would call it `itests` directly for integration tests.
```suggestion
members = [".", "ipld/resolver", "testing/e2e", "identity", "testing/itest"]
```